<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Presentation Maker</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTool_bars">
    <property name="title">
     <string>Tool bars</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <widget class="QMenu" name="menuDock_Widgets">
    <property name="title">
     <string>Dock Widgets</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTool_bars"/>
   <addaction name="menuDock_Widgets"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/New_Icon.ico</normaloff>:/Icons/New_Icon.ico</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="statusTip">
    <string>Create new project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Open_Icon.ico</normaloff>:/Icons/Open_Icon.ico</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="statusTip">
    <string>Open an existing file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Save_Icon.ico</normaloff>:/Icons/Save_Icon.ico</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="statusTip">
    <string>Save the project to disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Print_Icon.ico</normaloff>:/Icons/Print_Icon.ico</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="statusTip">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Exit_Icon.ico</normaloff>:/Icons/Exit_Icon.ico</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Cut_Icon.ico</normaloff>:/Icons/Cut_Icon.ico</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="statusTip">
    <string>Cut the current selection's contents to the clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Copy_Icon.ico</normaloff>:/Icons/Copy_Icon.ico</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy the current selection's contents to the clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Paste_Icon.ico</normaloff>:/Icons/Paste_Icon.ico</iconset>
   </property>
   <property name="text">
    <string>Past</string>
   </property>
   <property name="statusTip">
    <string>Paste the clipboard's contents into the current selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>Show the application's About box</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
   <property name="statusTip">
    <string>Show the Qt library's About box</string>
   </property>
  </action>
  <action name="actionDelete_Item">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/delete.ico</normaloff>:/Icons/delete.ico</iconset>
   </property>
   <property name="text">
    <string>Delete Item</string>
   </property>
   <property name="statusTip">
    <string>Delete item from diagram</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionBring_To_Front">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/bringtofront.ico</normaloff>:/Icons/bringtofront.ico</iconset>
   </property>
   <property name="text">
    <string>Bring To Front</string>
   </property>
   <property name="statusTip">
    <string>Bring item to front</string>
   </property>
  </action>
  <action name="actionSend_Item_To_Back">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/sendtoback.ico</normaloff>:/Icons/sendtoback.ico</iconset>
   </property>
   <property name="text">
    <string>Send Item To Back</string>
   </property>
   <property name="statusTip">
    <string>Send item to back</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
